packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -189,18 +205,47 @@ export default class MeetingInfoV2 {
     if (directURI) options.directURI = directURI;
 
     return this.webex.request(options)
+      .then(() => {
```

Comment:
if V1 is present its the new meetinginfo !?

Reply:
@arun3528, This fetches meeting info from the server, correct as discussed earlier, if MEETING_INFO_V1_SUCCESS it's the new meeting info